go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/godbus/dbus/v5 v5.0.3
 	github.com/muka/go-bluetooth v0.0.0-20200619025933-f6113f7141c5
 	github.com/sirupsen/logrus v1.6.0 // indirect
+	github.com/tinygo-org/bluetooth v0.0.0-20200801163934-de824e188486
```

Comment:
I think this import is here because of the `github.com/tinygo-org/bluetooth/winbt` import. I think that import should also be moved over to the custom import path.

Reply:
Good catch, just added a commit to handle that.